### ISSUE TITLE
Broken executable when building 2.1.5

### ISSUE THREAD

**Issue by Ambrevar** [monochromegane/the_platinum_searcher#180]
Cloning the repo, then running
```
git checkout v2.1.5
go get github.com/monochromegane/the_platinum_searcher/...
```
will result in a broken executable. (No help message, nothing.)
Am I doing something wrong?

**Comment by panchoh** [comment]
It seems that the next commit after the one from the release v2.1.5 fixes the issue.

**Comment by Ambrevar** [comment]
fff19c50638e4203f4cf0593d0067d97429b5101? It's only a change in wercker.yml, I don't see how it would change anything. Any clue?

Isn't it an issue with a dependency?

**Comment by panchoh** [comment]
Sorry, I wasn't explicit enough.  I meant c7d8eec66dca50773e6b4ee7dfdad2174860b9b1.  That's the one that fixes it.

**Comment by monochromegane** [comment]
Hi, I have released [v2.1.6](https://github.com/monochromegane/the_platinum_searcher/releases/tag/v2.1.6).
